new-module/flash-loans.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Flash loans using agTokens
+---
+
+# ⚡️ Angle flash-loans
+
+Flash-loans are now possible with agTokens!
+
+## What are flash loans? 
+
+They are a type of loans that allow to borrow and repay tokens in one transaction. By doing so, the entity taking out the loan can do so without putting down any collateral, as the lender is guaranteed to be paid back if the transaction is executed. 
```

Comment:
Let's insist on the fact that with Angle stablecoins are minted from the transaction and that volume can potentially be huge